### PR TITLE
fuzz: Add option to merge input dir to test runner

### PR DESCRIPTION
**Pull request by maflcko**
This is mainly useful for myself to merge pull requests like https://github.com/bitcoin-core/qa-assets/pull/4

I thought it wouldn't hurt to share the code.

Also remove the `--disable-ccache` from the docs to speed up builds when developing fuzzers.

### CODE REVIEW

**Comment by practicalswift** [comment]
Concept ACK

Nice to have!

**Comment by MarcoFalke** [comment]
re-run ci

File: test/fuzz/test_runner.py [hunk 1]
```diff
@@ -121,6 +136,22 @@ def main():
     )
 
 
+def merge_inputs(*, corpus, test_list, build_dir, merge_dir):
+    logging.info("Merge the inputs in the passed dir into the seed_dir. Passed dir {}".format(merge_dir))
```
Comment (sipa) [inline test/fuzz/test_runner.py]: This looks like leftover debugging.

**Review by sipa — COMMENTED** [review]
Concept ACK

File: test/fuzz/test_runner.py (hunk 1 above)
Comment (MarcoFalke) [reply test/fuzz/test_runner.py]: Merging is currently a manual operation, so I like the extra verbosity. Happy to remove if you have strong feelings about this.

Comment (sipa) [reply test/fuzz/test_runner.py]: If it's intentional, no problem; It just seemed written strangely as a log message, so I wanted to make sure it wasn't a leftover.

**Pull request merged**